community/releases/release-manager-runbook/index.md

```diff
@@ -43,13 +55,23 @@ make the required fixes to allow them to merge.
 1. Start with a [green build on master](https://builds.spinnaker.io/job/Flow_BuildAndValidate/).
 
 1. Create the release branches by running the [**Admin_StartReleaseBranch**](https://builds.spinnaker.io/job/Admin_StartReleaseBranch/build?delay=0sec)
-job:
-    
+job which will create `latest-unvalidated` when it passes:
+
     1. Set **NEW_BRANCH_NAME** to `${RELEASE_BRANCH}` (e.g., `release-1.20.x`).
-    
+
     1. Set **BASE_BRANCH** to `master`.
 
-1. Deactivate the now-oldest Flow_BuildAndValidate_* flow:
+    - you can check the status of each service build by going to [GCP cloudbuild](https://console.cloud.google.com/cloud-build/builds?project=spinnaker-community). Turn on the tags to see which each service name
+        <details>
```

Comment:
How is this going to render? I try to preview it, and it looks like everything in the < details > element is ignored.

Reply:
So meta. A gif within a gif XD
<kbd>![](https://p-qKFvWn.b3.n0.cdn.getcloudapp.com/items/mXuAJZz4/Screen%20Recording%202020-06-26%20at%2012.47.gif?v=84d2bf2793e5e969081ea741b0431df3)</kbd>

Reply:
💥